### ISSUE TITLE
IO doesn't load if mixer file has 8 outputs

### ISSUE THREAD

**Issue by AndreasAntener** [PX4/Firmware#2113]
Maybe related to https://github.com/PX4/Firmware/issues/2093

IO fails if the mixer has 8 outputs assigned, this happens on start:
```
[px4io] mixer send error -1
mixer: error loading mixers from /etc/mixers/AERT.main.mix: Operation not permitted
[i] Error loading mixer: /etc/mixers/AERT.main.mix
```

**Comment by AndreasAntener** [comment]
I have to add that although it seems to be calibrated, I get these messages now:
```
[07:41:38.102 - COMP:50] Critical: PREFLIGHT FAIL: ACCEL 0 SELFTEST FAILED
[07:41:38.166 - COMP:50] Critical: PREFLIGHT FAIL: ACCEL 1 SELFTEST FAILED
```

**Comment by AndreasAntener** [comment]
@LorenzMeier I'm pretty sure that this commit broke it: https://github.com/PX4/Firmware/commit/f8c8876642857520e85ec8ca027146c11b4301e3
It works before that.

**Comment by LorenzMeier** [comment]
@AndreasAntener Could you just revert the commit on master and verify that fixes it?

**Comment by AndreasAntener** [comment]
I just did that, it fixed the issue at startup. But I would like to arm and verify correct operation, but having trouble getting through preflight checks now ;)

**Comment by AndreasAntener** [comment]
Ok, 8 channels working: #2130

**Comment by AndreasAntener** [comment]
Fixed by #2130

**Comment by severinleuenberger** [comment]
double-checked - the 8ch *.mix does work again! 
Thanks
